packages/web-components/fast-foundation/src/tooltip/README.md

```diff
@@ -10,14 +10,24 @@ The `fast-tooltip` component is used provide extra information about another ele
 ## Usage
 
 ```html live
```

Comment:
should `html` be `tsx`?

Reply:
Indeedy, corrected this same thing in data grid sample as well.